src/common/state/RenderingAttributes.h

```diff
@@ -9,7 +9,7 @@
 #include <AttributeSubject.h>
 
 #include <ColorAttribute.h>
-#include <float.h>
+#include <visit-config.h>
 
```

Comment:
It was added there because many other files include it which then have #ifdef HAVE_OSPRAY checks.

For instance,  avt/VisWindow/Colleagues/VisWinRendering.C

which might not have #include <visit-config.h> but use rendering atts.  I was also finding that some files via other includes would have it but not others so it was also an attempt to make sure it was there regardless.

Reply:
I that visit-config.h should be included where it is needed, not by transitive inclusion.
HAVE_OSPRAY isn't used in this header, so isn't needed here.